__tests__/integration.test.js

```diff
@@ -103,6 +103,54 @@ describe("/api/articles/:article_id", () => {
     });
 });
 
+describe("/api/articles/:article_id/comments", () => {
+    test("GET:200 Should send an array of comments for the given article_id", () => {
+        return request(app)
+            .get("/api/articles/9/comments")
+            .expect(200)
+            .then((response) => {
+                const comments = response.body.comments;
+                expect(comments.length).toBe(2);
+                expect(comments).toBeSortedBy("created_at", {
```

Comment:
I would separate this sorted into a separate test - its not in your test description here